app/admin/RTMediaAdmin.php

```diff
@@ -445,29 +458,33 @@ public function rtmedia_addon_update_notice() {
 				}
 				rtmedia_update_site_option( 'rtmedia-addon-update-notice-3_8', 'show' );
 				?>
-				<div class="error rtmedia-addon-upate-notice">
+				<div class="error notice is-dismissible rtmedia-addon-update-notice">
 					<p>
-						<strong><?php esc_html_e( 'rtMedia:', 'buddypress-media' ) ?></strong>
-						<?php esc_html_e( 'Please update all premium add-ons that you have purchased from', 'buddypress-media' ) ?>
-						<a href="https://rtmedia.io/my-account/"
-						   target="_blank"><?php esc_html_e( 'your account', 'buddypress-media' ) ?></a>.
-						<a href="#" onclick="rtmedia_hide_addon_update_notice()"
-						   style="float:right"><?php esc_html_e( 'Dismiss', 'buddypress-media' ) ?></a>
+						<strong><?php esc_html_e( 'rtMedia:', 'buddypress-media' ); ?></strong>
+						<?php esc_html_e( 'Please update all premium add-ons that you have purchased from', 'buddypress-media' ); ?>
+						<a href="https://rtmedia.io/my-account/" target="_blank"><?php esc_html_e( 'your account', 'buddypress-media' ); ?></a>.
 						<?php wp_nonce_field( 'rtmedia-addon-update-notice-3_8', 'rtmedia-addon-notice' ); ?>
 					</p>
 				</div>
 				<script type="text/javascript">
-					function rtmedia_hide_addon_update_notice() {
-						var data = {
-							action: 'rtmedia_hide_addon_update_notice',
-							_rtm_nonce: jQuery('#rtmedia-addon-notice').val();
-					};
-						jQuery.post(ajaxurl, data, function (response) {
-							response = response.trim();
-							if (response === "1")
-								jQuery('.rtmedia-addon-upate-notice').remove();
-						});
-					}
+					jQuery( document ).ready( function () {
+
+						jQuery( '.rtmedia-addon-update-notice.is-dismissible' ).on( 'click', '.notice-dismiss', function () {
```

Comment:
Can you please consider creating a common functionality with this.

```<div class="error notice is-dismissible" data-action="the_action" data-nonce="nonce">Hello</div>```

in jquery
```
$(document).on('click', '.is-dismissible', function(){
    	var action = $(this).data('action');
        var nonce = $(this).data('nonce');
        alert(action);
        alert(nonce);
//further action....
    });

```
now next time you need dismissal notice you don't need to write a js code
just add action and nonce